phalcon/dispatcher.zep

```diff
@@ -387,6 +387,14 @@ abstract class Dispatcher implements DispatcherInterface, InjectionAwareInterfac
 	}
 
 	/**
+	 * Gets model binder
+	 */
+	public function getModelBinder() -> <BinderInterface>
+	{
+		return this->_modelBinder;
+	}
```

Comment:
There is many methods in whole framework like getter which have only class returned without `|null`.